draft-ietf-mls-protocol.md

```diff
@@ -1268,6 +1268,23 @@ client presents a new credential, the application MUST verify that the set
 of identifiers in the credential is acceptable to the application for this
 client.
 
+### Deniable Signature Keys
+
+Some 1-to-1 messaging protocols such as Signal achieve varying degrees of
+plausible deniability by leveraging the symmetry of the Diffie-Hellman key
+exchange.
+
+MLS uses signatures to achieve strong authentication of messages (see Section
+{{content-authentication}}). Since these signatures do not allow for plausible
+deniability, signatures on MLS messages can either be performed using the
+private key corresponding to the signature public key in the sender's
+credential, or by using another signature key that is not directly tied to the
+identity of the sender, thus allowing for some degree of deniability.
+
+Such deniable signature keys MUST be distributed and authenticated to and by
+each member out-of-band prior to any use for signature verification to ensure
+that MLS messages are still strongly authenticated.
```

Comment:
Given that this mostly has to do with how the AS works (and therefore it's out-of-scope for the spec), it might be more concise and also sufficient to say that signature keys must be authenticated by the AS, and cite the distribution of signature keys over a deniable channel as an example.